client/cmd/client/.env

```diff
@@ -2,3 +2,9 @@
 GRPC_HOST="localhost"
 # GRPC server port
 GRPC_PORT="8085"
+# Client certificate for tls secured connection with server.
```

Comment:
why .env file and not config.toml?

Reply:
Several reasons:
- for small projects or simple setups, using a single .env file can be straightforward and easy to manage.
- docker Compatibility: .env files are commonly used in Docker containers, and they integrate well with container orchestration tools like Docker Compose and Kubernetes. They allow for easy configuration changes without modifying the application code, which is very useful.

There is already a task created to make it runnable with either .env/configs/flags.